feature-libs/asm/customer-360/components/asm-customer-360/asm-customer-360.component.ts

```diff
@@ -38,18 +42,38 @@ import {
   DirectionMode,
   DirectionService,
   FocusConfig,
+  FocusDirective,
   ICON_TYPE,
+  IconComponent,
   LaunchDialogService,
+  MediaComponent,
+  MessageComponent,
 } from '@spartacus/storefront';
 import { BehaviorSubject, Observable, Subscription, of } from 'rxjs';
 import { catchError, distinctUntilChanged, filter, map } from 'rxjs/operators';
 import { AsmCustomer360Config } from '../config/asm-customer-360-config';
+import { AsmCustomer360SectionComponent } from '../sections/asm-customer-360-section/asm-customer-360-section.component';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'cx-asm-customer-360',
   templateUrl: './asm-customer-360.component.html',
-  standalone: false,
+  imports: [
+    FocusDirective,
+    NgTemplateOutlet,
+    NgIf,
+    MessageComponent,
+    MediaComponent,
+    IconComponent,
+    NgFor,
+    AsmCustomer360SectionComponent,
+    AsyncPipe,
+    TranslatePipe,
+    CxDatePipe,
+    ArgsPipe,
+
+    CxDatePipe,
```

Comment:
`minor:` let's remove duplicated import.

Tis comment applies to more places (e.g. [order-overview.component.ts](https://github.com/SAP/spartacus/blob/a615077b9172e79a1e116333d17bc415a25d9af1/feature-libs/order/components/order-details/order-overview/order-overview.component.ts#L52-L54))